source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,23 @@
+// start-specify-v1
+<?php
+
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = [
+    'serverApi' => new MongoDB\Driver\ServerApi('1')
+];
+
+$client = new MongoDB\Client($uri, [], $driverOptions);
+// end-specify-v1
+
+// start-stable-api-options
+<?php
```

Comment:
I don't recall seeing multiple `<?php` tags in source files from @norareidy's previous PRs. This makes the file itself invalid, unless you intend to add a `?>` above (probably between `end-specify-v1` and `start-stable-api-options` since there's no reason to show `?>` to a user (they won't use it in their files).

If these source files are ever meant to be executed for correctness (as we do for some existing sources in the PHPLIB docs before the docs team took over), that'd be even more reason to ensure the syntax within is valid.

Reply:
i wasn't sure about these. the existing, un-standardized docs (e.g., hhttps://www.mongodb.com/docs/php-library/current/tutorial/gridfs/#uploading-files-with-writable-streamsere) include them in all code samples, so i assumed it was the preferred syntax.

these files will never be executed _in toto_, so i'm just going to remove these tags.

Reply:
> include them in all code samples

I think that was necessary for syntax highlighting. Note that most of those code blocks are embedded in the RST and weren't being included from a common file (nor do they get automated testing).

Speaking just of the [GridFS examples](https://www.mongodb.com/docs/php-library/current/tutorial/gridfs/), I'll also note that those are all full scripts that can be executed on their own (each constructs a Client).

I think it makes sense to leave `<?php` at the top of each of your files and won't hurt to show it in the lead-in example that constructs a client (or the sample app), but if you don't need it for syntax highlighting feel free to get rid of it.